### PR TITLE
typo

### PR DESCRIPTION
**Pull request by freakin23**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

Issue Linked: #4787

### CODE REVIEW

**Comment by freakin23** [comment]
AC (Python): https://cses.fi/paste/716cf06babe15629a14956/
AC (C++): https://cses.fi/paste/bfd94f4fa6f9ee7fa1492b/

**Comment by freakin23** [comment]
Implementation 2 AC (C++): https://cses.fi/paste/f81b53fd18962639a1483b/

**Comment by freakin23** [comment]
I added implementation 2 just in case, if it is unnecessary just tell me I will remove it.

File: solutions/gold/cses-3221.mdx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+id: cses-3221
+source: CSES
+title: Sliding Window Minimum
+author: Rameez Parwez
+---
+
+[Official Analysis] (https://cp-algorithms.com/data_structures/stack_queue_modification.html#queue-modification-method-1)
```
Comment (SansPapyrus683) [inline solutions/gold/cses-3221.mdx:8]: pretty sure this is unofficial

File: content/4_Gold/Sliding_Window.problems.json [hunk 2]
```diff
@@ -190,6 +190,18 @@
...
+      "name": "Sliding Window Minimum",
+      "url": "https://cses.fi/boi24/task/3221/",
+      "source": "CSES",
+      "difficulty": "Easy",
+      "isStarred": true,
+      "tags": [],
+      "solutionMetadata": {
+        "kind": "internal"
+      }
+    },
```
Comment (SansPapyrus683) [inline content/4_Gold/Sliding_Window.problems.json:204]: how's this different from the sliding window maximum problem

Comment (freakin23) [reply content/4_Gold/Sliding_Window.problems.json:204]: Nothing, it's just minimum

Comment (SansPapyrus683) [reply content/4_Gold/Sliding_Window.problems.json:204]: uh
not really sure why we should add this then

Comment (freakin23) [reply content/4_Gold/Sliding_Window.problems.json:204]: Just think like we not only solve "nearest greater element" prblm but also the "nearest smallest element".

Comment (SansPapyrus683) [reply content/4_Gold/Sliding_Window.problems.json:204]: i mean idk why that problem was added
sorry this just seems, really redundant

Comment (freakin23) [reply content/4_Gold/Sliding_Window.problems.json:204]: Okay if you say so

Comment (TheGamingMousse) [reply content/4_Gold/Sliding_Window.problems.json:204]: > Nothing, it's just minimum

so it's just changing min to max? the module has sliding window code already

**Comment by TheGamingMousse** [comment]
@freakin23 so i see that ur PR is a response to this

image

my question is, are you planning on removing the LC problem? cuz rn ur making it so we have two problems that are essentially identical

**Review by ryanchou-dev — COMMENTED** [review]
might be helpful to give a brief coverage of the deque method in the module

File: content/4_Gold/Sliding_Window.problems.json (hunk 2 above)
Comment (freakin23) [reply content/4_Gold/Sliding_Window.problems.json:204]: yes i will going to remove it

**Comment by freakin23** [comment]
> @freakin23 so i see that ur PR is a response to this
> 
> image
> 
> my question is, are you planning on removing the LC problem? cuz rn ur making it so we have two problems that are essentially identical

at first i was only paying attention to the subtopic and got some kind of misunderstanding. ig only changing the subtopic would be good.

**Comment by freakin23** [comment]
ig now it is good.

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**